docs/public/components/overview.md

```diff
@@ -5,68 +5,35 @@ is invoked, different components may be instantiated.
 
 Since each component is an interface to the outside we can have several implementations for it.
 
-## Grouping components: Bundles
-
-There is a large and growing number of components, and listing those components out repeatedly could grow tiresome and
-cause bugs. Component bundles provide a way to manipulate multiple components, usually at the meta or service level, as
-a single unit. For example, while Logs-Agent is internally composed of many components, those can be used as a single unit
-with a `logs` bundle.
-
-As an aid to thinking about this question, consider the following levels:
-
-1. Large-scale parts of the Agent that agglomerate multiple components would be a `Bundle`. Example: the Logs-Agent.
-2. Dedicated section or logic block that can be reused in multiple places would be a `Component`. Example: the logger
-   component.
-3. A specific version of a component would be an implementation. Example: a component in charge of compressing payload
-   could have a ZSTD and ZIP implementation. Most components will only have a single implementation.
-
-## Stateless logic
-
-Any logic/helper that is stateless (no globals and not `init` function) can but doesn't need to be a component. Such
-basic helpers can remain or be added to the `pkg` folder.
-
 ## FX vs Go module
 
-Components are designed to be used with a dependency injection. In the Agent repo we use [Fx](fx.md) for this. All agent
+Components are designed to be used with a dependency injection framework. In the Agent we use [Fx](fx.md) for this. All agent
 binaries use `FX` to load, coordinate and start the required components.
 
-Some components are used outside the `datadog-agent` repository where `FX` is not available. To support this, the
-`FX` wrappers are split from each component's implementations. Components implementation can be exported as a Go module. We
-will see in more detail how to create components in the next section.
+Some components are used outside the `datadog-agent` repository where `FX` is not available. To support this, the components implementation must not require `FX`. 
+Components implementation can be exported as a Go module. We will see in more detail how to create components in the next section.
 
 The important information here is that it's possible to use components without FX **outside** the agent repository. This
 comes at the cost of manually doing the work of `Fx`, our dependency injection framework.
 
 ## Important note on Fx
 
 The component framework project core goal is to improve the Agent codebase by decoupling parts of the code, removing global state and init
-methods, and increased reusability by separating logical units into components. We use the `Fx` library to perform
-dependency injection and startup orchestration of the agent components, however **`Fx` itself is not intrinsic to the
+methods, and increased reusability by separating logical units into components. **`Fx` itself is not intrinsic to the
```

Comment:
I think a quick explanation of what FX is used for was useful in this context, no ?

Reply:
I agree. Let's have a disclaimer about components not being 1 to 1 to Fx